mozci/task.py

```diff
@@ -265,35 +258,22 @@ def _load_errorsummary(self):
         ]
 
         self.__post_init__()
-        # Only store data if there's something to store
-        if self._errors or self._results:
-            logger.debug("Storing {} in the cache".format(self.id))
-            # cachy's put() overwrites the value in the cache; add() would only add if its empty
-            # Temporarily disable caching as it's creating too many entries.
-            # adr.config.cache.put(
-            #     self.id,
-            #     {
-            #         "errors": self._errors,
-            #         "results": self._results,
-            #     },
-            #     adr.config["cache"]["retention"],
-            # )
 
     @property
     def groups(self):
-        if self._results is None:
+        if not self._results:
```

Comment:
If _results (or _errors in errors) is a list with 0 elements, we'd be calling _load_errorsummary again. I don't remember exactly why you had to do this in a previous iteration of the patch, but maybe it's no longer needed now?

Reply:
I got confused with seeing `Fetching errorsummary for` for every single task.